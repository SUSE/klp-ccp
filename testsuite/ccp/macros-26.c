#define NO_RETAIN(x) #x, x

int espr_i;

void pu_f(void)
{
	NO_RETAIN(espr_i);
}
