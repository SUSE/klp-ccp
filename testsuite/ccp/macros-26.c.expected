/* klp-ccp: from testsuite/ccp/macros-26.c */
static int (*klpe_espr_i);

void klpp_pu_f(void)
{
	"espr_i", (*klpe_espr_i);
}
